Sources/Orbit/Components/Alert.swift

```diff
@@ -12,284 +12,195 @@ import SwiftUI
-                             .primaryAndSecondary(_, let secondaryButton):
-                            Button(secondaryButton.label, type: secondaryButtonType, action: secondaryButton.action)
-                                .accessibility(.alertButtonSecondary)
-                        case .none, .primary, .inline:
-                            EmptyView()
-                    }
-                }
-                .buttonSize(.compact)
-            case .none, .inline:
-                EmptyView()
+    /// Creates Orbit Alert component with custom content and icon.
+    init(
+        _ title: String = "",
+        description: String = "",
+        isSubtle: Bool = false,
+        @AlertButtonsBuilder buttons: () -> Buttons,
+        @ViewBuilder content: () -> Content = { EmptyView() },
+        @ViewBuilder icon: () -> Icon = { EmptyView() }
+    ) {
+        self.init(title: title, description: description, isSubtle: isSubtle) {
+            buttons()
+        } content: {
+            content()
+        } icon: {
+            icon()
         }
     }
+}
 
-    @ViewBuilder var background: some View {
-        backgroundColor
-            .overlay(
-                RoundedRectangle(cornerRadius: BorderRadius.default)
-                    .strokeBorder(strokeColor, lineWidth: 1)
-            )
-            .overlay(
-                outlineColor
-                    .frame(height: 3),
-                alignment: .top
-            )
-    }
+// MARK: - Types
+struct AlertButtonStyle: PrimitiveButtonStyle {
 
-    var outlineColor: Color {
-        switch style {
-            case .status(let status, _):        return (status ?? defaultStatus).color
-        }
-    }
-    
-    var backgroundColor: Color {
-        switch style {
-            case .status(_, true):              return .cloudLight
-            case .status(let status, false):    return (status ?? defaultStatus).lightColor
+    @Environment(\.status) private var status
+    @Environment(\.isSubtle) private var isSubtle
+
+    let isPrimary: Bool
+
+    public func makeBody(configuration: Configuration) -> some View {
+        OrbitCustomButtonContent(
+            configuration: configuration,
+            textColor: textColor,
+            horizontalPadding: .small,
+            verticalPadding: .xSmall, // = 32 height @ normal size
+            cornerRadius: BorderRadius.desktop,
+            hapticFeedback: resolvedStatus.defaultHapticFeedback
+        ) {
+            EmptyView()
+        } disclosureIcon: {
+            EmptyView()
+        } background: {
+            background
+        } backgroundActive: {
+            backgroundActive
         }
+        .textFontWeight(.medium)
+        .textSize(.small)
     }
 
-    var strokeColor: Color {
-        switch style {
-            case .status(_, true):              return .cloudLightHover
-            case .status(let status, false):    return (status ?? defaultStatus).lightHoverColor
+    var textColor: Color {
```

Comment:
A `textColorActive` should be also added, using the `DarkActive` variant of color.

Reply:
Suggestion - we could replace the textColor parameter in `OrbitCustomButtonContent ` with environment key - after all, the symmetry with active variant is not better than duplicating the property with existing key 🙏

Reply:
Updated.